### ISSUE TITLE
sequencer (for chords)

### ISSUE THREAD

**Issue by ghost** [bhostetler18/tonality_features#12]
This might be overkill but the app itself is so broad that there should definitely be place for it. For the chord pads something like a "song mode" /sequence where you can program chords to repeat a X number of beats and/or bars.

For reference I would look at the song mode of DM2, the sequence(pattern) mode of the Roseta suit maybe even Chordbud app.

Even better it would be to be able to group sequences to avoid repetitions (think of grouping a song sections AABA, for instance, where A, B, etc is a sequence of chords).

I write this as my main use for Tonality is the chord pads in AUv3 mode.

I rarely use the other features.

**Comment by bhostetler18** [comment]
@damoga Yeah I've definitely been considering this. It's a long term goal as I have a lot of other feature requests and ideas. 

Have you used the automation/navigation parameters? Some clever application could allow you to achieve a similar effect. I'd happily add additional parameters like section markers to make this more feasible.

**Comment by ghost** [comment]
It's ok, I'll have a look at the automation parameters.
So much to explore here!
